app/views/jst/courses/roster/rosterUser.handlebars

```diff
@@ -2,7 +2,11 @@
   {{>avatar}}
 </td>
 <td>
-  <a href="{{html_url}}" class="roster_user_name">{{name}}</a>
+  {{#if canViewSisId}}
+  <a href="{{html_url}}" class="roster_user_name">{{name}}</a> ({{short_name}})
```

Comment:
Yeah, maybe this could really use a new `roster_user_name` display variable that's already built in the API.